cdc/sink/common/common.go

```diff
@@ -83,10 +83,8 @@ func (c *UnresolvedTxnCache) Append(filter *filter.Filter, rows ...*model.RowCha
 			// fail-fast check
 			if len(txns) != 0 && txns[len(txns)-1].commitTs > row.CommitTs {
 				log.Panic("the commitTs of the emit row is less than the received row",
-					zap.Stringer("table", row.Table),
-					zap.Uint64("emit row startTs", row.StartTs),
-					zap.Uint64("emit row commitTs", row.CommitTs),
-					zap.Uint64("last received row commitTs", txns[len(txns)-1].commitTs))
+					zap.Uint64("last received row commitTs", txns[len(txns)-1].commitTs),
```

Comment:
```suggestion
					zap.Uint64("lastReceivedCommitTs", txns[len(txns)-1].commitTs),
```

We prefer camelCase.